src/main/java/com/technopolis/server/server/model/User.java

```diff
@@ -0,0 +1,33 @@
+package com.technopolis.server.server.model;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+@Table(name = "users")
+@Data
+public class User extends BaseEntity {
+    @Column(name = "username")
```

Comment:
Данное поле не должно быть null, поэтому код должен быть следующим:
```suggestion
    @Column(name = "username", nullable = false)
```